linera-views/src/dynamo_db.rs

```diff
@@ -322,7 +291,17 @@ impl DynamoDbBatch {
     }
 
     fn is_fastpath_feasible(&self) -> bool {
-        self.len() <= MAX_TRANSACT_WRITE_ITEM_SIZE
+        if self.len() > MAX_TRANSACT_WRITE_ITEM_SIZE {
+            return false;
+        }
+        let mut total_size = 0;
+        for (key, value) in &self.0.insertions {
+            total_size += key.len() + value.len();
+        }
+        for deletion in &self.0.deletions {
+            total_size += deletion.len();
```

Comment:
Do we need to add 2 resp. 1 here, too, to account for the BCS encoding?

Reply:
Yes, I did some separate checks.